com.dudetronics.DroidStar.desktop

```diff
@@ -0,0 +1,7 @@
+[Desktop Entry]
+Exec=com.dudetronics.DroidStar
+Icon=com.dudetronics.DroidStar
+Name=com.dudetronics.DroidStar
```

Comment:
I doubt that "name" and "exec" are the right one. I only mentionned the icon name.